-------------------------------------------------------------------------------
Test set: io.apicurio.tests.smokeTests.apicurio.MetadataIT
-------------------------------------------------------------------------------
Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 2.954 s <<< FAILURE! -- in io.apicurio.tests.smokeTests.apicurio.MetadataIT
io.apicurio.tests.smokeTests.apicurio.MetadataIT.getAndUpdateMetadataOfArtifact -- Time elapsed: 0.041 s <<< ERROR!
java.lang.RuntimeException: java.util.concurrent.ExecutionException: io.vertx.core.http.HttpClosedException: Connection was closed
	at io.kiota.http.vertx.VertXRequestAdapter.getHttpResponseMessage(VertXRequestAdapter.java:437)
	at io.kiota.http.vertx.VertXRequestAdapter.send(VertXRequestAdapter.java:167)
	at jdk.internal.reflect.GeneratedMethodAccessor33.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at io.apicurio.registry.client.RegistryClientFactory$RetryInvocationHandler.invoke(RegistryClientFactory.java:146)
	at jdk.proxy2/jdk.proxy2.$Proxy65.send(Unknown Source)
	at io.apicurio.registry.rest.client.groups.item.artifacts.ArtifactsRequestBuilder.post(ArtifactsRequestBuilder.java:130)
	at io.apicurio.tests.ApicurioRegistryBaseIT.createArtifact(ApicurioRegistryBaseIT.java:194)
	at io.apicurio.tests.smokeTests.apicurio.MetadataIT.getAndUpdateMetadataOfArtifact(MetadataIT.java:36)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.util.concurrent.ExecutionException: io.vertx.core.http.HttpClosedException: Connection was closed
	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073)
	at io.kiota.http.vertx.VertXRequestAdapter.getHttpResponseMessage(VertXRequestAdapter.java:431)
	... 12 more
Caused by: io.vertx.core.http.HttpClosedException: Connection was closed

